centreon/unattended.sh

```diff
@@ -396,11 +395,9 @@ function set_required_prerequisite() {
 	    ;;
         esac
 
-        install_remi_repo
```

Comment:
Could you also delete the function ?
https://github.com/centreon/centreon/blob/2219f1adbd1ff9e0dcfa480f0b625afaf104cb77/centreon/unattended.sh#L535

Reply:
this function is used for el8 install